docs/bugs_and_glitches.md

```diff
@@ -1738,6 +1739,39 @@ Most of the NPCs in [maps/NationalParkBugContest.asm](https://github.com/pret/po
 (The use of `SPRITE_ROCKER` instead of `SPRITE_COOLTRAINER_M` for `COOLTRAINERM NICK` may also be an intentional reference to the player's brother from the [Space World '97 beta](https://github.com/pret/pokegold-spaceworld).)
 
 
+### A hatching Unown egg would not show the right letter
+
+This happens because both `GetEggFrontpic` and `GetHatchlingFrontpic` use `wBattleMonDVs`, but that's not initialized. They should use the `wCurPartyMon`th `wPartyMon#DVs` instead.
```

Comment:
How about "They should use the current party mon's DVs instead." since `wCurPartyMon` and `wPartyMon#DVs` do not appear directly in the fix. (One can look at the code of `GetPartyParamLocation` to see how it works.)

Reply:
I was also thinking of doing this:

```
ld a, [wCurPartyMon]
ld hl, wPartyMon1DVs
call GetPartyLocation
```

I prefer the former, but this one might help make things clearer.

Reply:
`GetPartyParamLocation` exists already and makes the code shorter, so I agree it's best to use `GetPartyParamLocation` when possible.

Reply:
Mmm. Fair.